lib/sirius_api/strategies/remote_oauth_server.rb

```diff
@@ -51,38 +47,42 @@ def access_token
       end
 
       def request_token_info(token_value)
-        resp = http_client.get(CHECK_TOKEN_URI, token: token_value)
+        resp = http_client.post(CHECK_TOKEN_URI, token: token_value)
         OpenStruct.new(resp.body).tap do |s|
           s.status = resp.status
         end
       end
 
-      def validate_token_info(token)
-        if token.status == 404
-          "Invalid OAuth access token."
+      def validate_token(token)
+        if token.status == 400
+          "Invalid access token."
         elsif token.status != 200
-          "Unable to verify OAuth access token (#{token.status})."
-        elsif token.client_id.blank?
-          "Invalid response from the OAuth authorization server."
+          "Unable to verify access token (#{token.status})."
```

Comment:
It would be nice to clarify the numeric code here (e.g. `(auth server response status: #{token.status})`)